include/language/en_us.lang.php

```diff
@@ -148,122 +148,132 @@
     'opportunity_relationship_type_dom' => array(
-            '' => '',
-            'Primary Decision Maker' => 'Primary Decision Maker',
-            'Business Decision Maker' => 'Business Decision Maker',
-            'Business Evaluator' => 'Business Evaluator',
-            'Technical Decision Maker' => 'Technical Decision Maker',
-            'Technical Evaluator' => 'Technical Evaluator',
-            'Executive Sponsor' => 'Executive Sponsor',
-            'Influencer' => 'Influencer',
-            'Other' => 'Other',
-        ),
+        '' => '',
+        'Primary Decision Maker' => 'Primary Decision Maker',
+        'Business Decision Maker' => 'Business Decision Maker',
+        'Business Evaluator' => 'Business Evaluator',
+        'Technical Decision Maker' => 'Technical Decision Maker',
+        'Technical Evaluator' => 'Technical Evaluator',
+        'Executive Sponsor' => 'Executive Sponsor',
+        'Influencer' => 'Influencer',
+        'Other' => 'Other',
+    ),
     //Note:  do not translate case_relationship_type_default_key
 //       it is the key for the default case_relationship_type_dom value
     'case_relationship_type_default_key' => 'Primary Contact',
     'case_relationship_type_dom' => array(
-            '' => '',
-            'Primary Contact' => 'Primary Contact',
-            'Alternate Contact' => 'Alternate Contact',
-        ),
+        '' => '',
+        'Primary Contact' => 'Primary Contact',
+        'Alternate Contact' => 'Alternate Contact',
+    ),
     'payment_terms' => array(
-            '' => '',
-            'Net 15' => 'Net 15',
-            'Net 30' => 'Net 30',
-        ),
+        '' => '',
+        'Net 15' => 'Net 15',
+        'Net 30' => 'Net 30',
+    ),
     'sales_stage_default_key' => 'Prospecting',
     'sales_stage_dom' => array(
-            'Prospecting' => 'Prospecting',
-            'Qualification' => 'Qualification',
-            'Needs Analysis' => 'Needs Analysis',
-            'Value Proposition' => 'Value Proposition',
-            'Id. Decision Makers' => 'Identifying Decision Makers',
-            'Perception Analysis' => 'Perception Analysis',
-            'Proposal/Price Quote' => 'Proposal/Price Quote',
-            'Negotiation/Review' => 'Negotiation/Review',
-            'Closed Won' => 'Closed Won',
-            'Closed Lost' => 'Closed Lost',
-        ),
+        'Prospecting' => 'Prospecting',
+        'Qualification' => 'Qualification',
+        'Needs Analysis' => 'Needs Analysis',
+        'Value Proposition' => 'Value Proposition',
+        'Id. Decision Makers' => 'Identifying Decision Makers',
+        'Perception Analysis' => 'Perception Analysis',
+        'Proposal/Price Quote' => 'Proposal/Price Quote',
+        'Negotiation/Review' => 'Negotiation/Review',
+        'Closed Won' => 'Closed Won',
+        'Closed Lost' => 'Closed Lost',
+    ),
+    'in_total_group_stages' => array(
```

Comment:
To avoid any mistakes in the future `sales_probability_dom` should be next to `sales_stage_dom` as keys must be the same as sales_stage_dom